torchaudio/csrc/ffmpeg/stream_reader/stream_reader.h

```diff
@@ -100,6 +100,12 @@ class StreamReader {
   // Retrieval
   //////////////////////////////////////////////////////////////////////////////
   std::vector<c10::optional<torch::Tensor>> pop_chunks();
+
+ private:
+  //////////////////////////////////////////////////////////////////////////////
+  // Instance variables
+  //////////////////////////////////////////////////////////////////////////////
+  int64_t seek_timestamp = -1;
```

Comment:
Could you move this to the section at the top of class definition where the rest of other variables are defined?